cli/file_fetcher.rs

```diff
@@ -515,11 +515,29 @@ impl FileFetcher {
     }
   }
 
+  #[inline(always)]
+  pub async fn fetch_bypass_permissions(
+    &self,
+    specifier: &ModuleSpecifier,
+  ) -> Result<File, AnyError> {
+    self.fetch_inner(specifier, None).await
+  }
+
+  // todo(THIS PR): lol what... this is never used?
   /// Fetch a source file and asynchronously return it.
+  #[inline(always)]
   pub async fn fetch(
```

Comment:
This function was used in a lot of places that mostly fetched configuration files and used "all permissions"

Reply:
Yeah, it just seems we always fetched with all permissions and effectively didn't use what is now this code.